src/java/org/apache/cassandra/service/StartupChecks.java

```diff
@@ -187,6 +190,58 @@ public void verify(StartupChecksOptions options) throws StartupException
         }
     }
 
+    // https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1057843
+    public static final StartupCheck checkKernelBug1057843 = new StartupCheck()
```

Comment:
Well, I would not do that:) It is because those bugs may be very specific and come with various different checks other than just a kernel version check (just like here we have ext4 and direct io check). Also, they are intended to be disable individual one by one rather than altogether or none. I really hope we won't have to have more such checks though...

Reply:
Nothing wrong with how it is now ... if we end up adding multiple kernel checks and we will see a need to start to group them together, then we can refactor. But having a group of kernel checks while there is just one just is just not necessary.